capa/features/extractors/dnfile/helpers.py

```diff
@@ -131,10 +133,17 @@ def get_dotnet_managed_imports(pe: dnfile.dnPE) -> Iterator[DnType]:
             # remove get_/set_ from MemberRef name
             member_ref_name = member_ref_name[4:]
 
+        typeref_name = []
+        typerefname = (member_ref.Class.row.TypeName,)
+        typerefnamespace = member_ref.Class.row.TypeNamespace
```

Comment:
Can we move these declarations to an `else` clause that is attached to the `if` statement below? This avoids creating and then immediately overwriting these objects when nested classes are encountered.

Reply:
Good point, fixed this in all places where it occurs.